css/home-page.styl

```diff
@@ -439,6 +440,10 @@ $flex-section
           &:nth-child(odd)
             justify-content: flex-end
 
+        @media (max-width: 400px)
+          flex-basis: 100%
+          flex-direction: column
+
```

Comment:
Maybe CSS multi-column spec would work better than flex in this case? https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Columns/Using_multi-column_layouts